src/main/java/org/candlepin/subscriptions/db/model/TallySnapshot.java

```diff
@@ -130,4 +130,14 @@ public TallyGranularity getGranularity() {
     public void setGranularity(TallyGranularity granularity) {
         this.granularity = granularity;
     }
+
+    public org.candlepin.subscriptions.utilization.api.model.TallySnapshot asApiSnapshot() {
+        org.candlepin.subscriptions.utilization.api.model.TallySnapshot snapshot =
+            new org.candlepin.subscriptions.utilization.api.model.TallySnapshot();
+
+        snapshot.setDate(this.getSnapshotDate());
+        snapshot.setCores(this.getCores());
```

Comment:
Actually scrap this. This should already be done.

Reply:
We'll skip owner/account number for now since it's implied the frontend will know these by context. Good thought that we should keep in mind for future though. If we added, I'd add to the meta section since they're shared by all snapshots in a given response.

Reply:
:+1: